client/src/components/ConversationDialog.js

```diff
@@ -89,7 +90,8 @@ function ConversationDialog(props) {
         <DialogContent>
           <Container className={classes.root}>
             <Typography className={classes.title}>{title}</Typography>
-            <Paper className={classes.audio} />
+            <AudioPlayer audioURL={audioURL} />
+            <audio controls src={audioURL} />
```

Comment:
Here I get an error with the AudioPlayer component when trying to playback the audio:
`Access to XMLHttpRequest at 'https://twitter-audio.s3.amazonaws.com/1564103805507' from origin 'http://localhost:3000' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.`

Reply:
The audio can be played back fine using HTML5 <audio> tag

Reply:
I read maybe you have to allow CORS on s3? I am not sure why this is happening

Reply:
You have to add CORS to the bucket:
https://docs.aws.amazon.com/AmazonS3/latest/user-guide/add-cors-configuration.html